smithy-swift-codegen/src/test/kotlin/ContentMd5MiddlewareTests.kt

```diff
@@ -7,35 +7,7 @@ class ContentMd5MiddlewareTests {
         val context = setupTests("Isolated/contentmd5checksum.smithy", "aws.protocoltests.restxml#RestXml")
         val contents = getFileContents(context.manifest, "/RestXml/RestXmlProtocolClient.swift")
         val expectedContents = """
-    public func idempotencyTokenWithStructure(input: IdempotencyTokenWithStructureInput) async throws -> IdempotencyTokenWithStructureOutput {
```

Comment:
This test is ridiculously overbroad for "generates ContentMD5 middleware" so I cut out the expected Swift code unrelated to that test assertion.